lib/exogenesis.rb

```diff
@@ -10,6 +10,7 @@
 require 'exogenesis/passengers/git_repo'
 require 'exogenesis/passengers/npm'
 require 'exogenesis/passengers/shell'
+require 'exogenesis/passengers/vim_plug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.